internal/deprecations.go

```diff
@@ -97,6 +97,42 @@ func NormalizeClusterSpec(cluster *fdbv1beta2.FoundationDBCluster, options Depre
 		updateImageConfigs(&cluster.Spec, cluster.GetUseUnifiedImage())
 	}
 
+	if cluster.Spec.Buggify.CrashLoop != nil && len(cluster.Spec.Buggify.CrashLoop) > 0 {
+		crashLoopContainers := cluster.GetCrashLoopContainerProcessGroups()
+		if _, ok := crashLoopContainers[fdbv1beta2.MainContainerName]; !ok {
+			crashLoopContainers[fdbv1beta2.MainContainerName] = make(map[string]fdbv1beta2.None)
+			crashLoopContainers[fdbv1beta2.SidecarContainerName] = make(map[string]fdbv1beta2.None)
+		}
+
+		for _, pid := range cluster.Spec.Buggify.CrashLoop {
+			if pid == "*" {
+				for _, proc := range cluster.Status.ProcessGroups {
+					crashLoopContainers[fdbv1beta2.MainContainerName][proc.ProcessGroupID] = fdbv1beta2.None{}
+					crashLoopContainers[fdbv1beta2.SidecarContainerName][proc.ProcessGroupID] = fdbv1beta2.None{}
```

Comment:
```suggestion
					crashLoopContainers[fdbv1beta2.MainContainerName][proc.ProcessGroupID] = fdbv1beta2.None{}
```

The old behaviour was to only put the main container into crash loop and we should keep it like that for compatibility.